tss-esapi/src/lib.rs

```diff
@@ -28,6 +28,7 @@
     unused_results,
     missing_copy_implementations
 )]
+#![allow(clippy::upper_case_acronyms)]
```

Comment:
Could we add a separate commit to fix the acronyms?

Reply:
Done

Reply:
I was thinking about actually replacing the acronyms with snake case but that's good for now!

Reply:
I think this "separate branch for 5.x.y" releases is going to be painful. I mean, if I replace the acronyms, that change needs to go into `main`, not in `5.x.y`. I still need to allow the lint on the `5.x.y` branch.

Also this is an example of why having alternative commits won't work - if I fix the acronyms in the `main` branch and then do a "non-breaking" commit where I use one of the acronyms somewhere, if I make that commit on top of `main` I'll be using the new version of the acronyms, whereas if I cherry-pick in `5.x.y` the commit I need it to be using the current version